geniza/corpus/metadata_export.py

```diff
@@ -165,3 +159,21 @@ def get_export_data_dict(self, doc):
         outd["has_translation"] = doc.has_translation()
 
         return outd
+
+
+class AdminDocumentExporter(DocumentExporter):
+    def get_export_data_dict(self, doc):
+        outd = super().get_export_data_dict(doc)
+        outd["notes"] = doc.notes
+        outd["needs_review"] = doc.needs_review
+        outd["status"] = doc.get_status_display()
+        outd[
+            "url_admin"
+        ] = f"{self.url_scheme}{self.site_domain}/admin/corpus/document/{doc.id}/change/"
+
+        return outd
+
+
+class PublicDocumentExporter(DocumentExporter):
+    def get_queryset(self):
```

Comment:
Do we need to subset the list of csv fields here, or does it happen automagically/implicitly because we're using csv `DictWriter`? 

If so, that seems slick but probably worth a comment noting that fact

Reply:
This is so automagic since the keys aren't populated in the dictionary csv DictWriter receives. Will add a note